rpcs3/rpcs3qt/settings_dialog.cpp

```diff
@@ -401,6 +401,12 @@ settings_dialog::settings_dialog(std::shared_ptr<gui_settings> xSettings, const
 	xemu_settings->EnhanceCheckBox(ui->stretchToDisplayArea, emu_settings::StretchToDisplayArea);
 	ui->stretchToDisplayArea->setToolTip(json_gpu_main["stretchToDisplayArea"].toString());
 
+	xemu_settings->EnhanceCheckBox(ui->scrictModeRendering, emu_settings::StrictRenderingMode);
+	ui->scrictModeRendering->setToolTip(json_gpu_main["scrictModeRendering"].toString());
+
+	xemu_settings->EnhanceCheckBox(ui->disableVertexCache, emu_settings::DisableVertexCache);
+	ui->scrictModeRendering->setToolTip(json_gpu_main["disableVertexCache"].toString());
```

Comment:
I think should be 

ui->disableVertexCache->setToolTip(json_gpu_main["disableVertexCache"].toString());